packages/p2p/src/components/buy-sell/__tests__/buy-sell-row.spec.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { fireEvent, screen, render } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import { useStores } from 'Stores';
+import BuySellRow from '../buy-sell-row.jsx';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+jest.mock('@deriv/shared/src/utils/screen/responsive', () => ({
+    ...jest.requireActual('@deriv/shared/src/utils/screen/responsive'),
+    isMobile: jest.fn(),
+}));
+
+const setSelectedAdvert = jest.fn();
+const showAdvertiserPage = jest.fn();
+
+const mockProps = () => ({
+    row: {
+        advertiser_details: {
+            id: '',
+            name: 'Test',
+        },
+        id: '',
+    },
+});
+
```

Comment:
@artsemkavaleu-deriv maybe it will be better to use:
const props = { your props } instead of invoke mockProps function in every test case,
and when you want to change props (for example props.row.id = 'NO_MATCH_ROW' (string 95))
you can write:
render(<BuySellRow {...props} row={ ...props.row, id: 'NO_MATCH_ROW' } />);

Reply:
> @artsemkavaleu-deriv maybe it will be better to use: const props = { your props } instead of invoke mockProps function in every test case, and when you want to change props (for example props.row.id = 'NO_MATCH_ROW' (string 95)) you can write: render(<BuySellRow {...props} row={ ...props.row, id: 'NO_MATCH_ROW' } />);

it's done, thx for the suggestion!